bitnami/harbor/values.yaml

```diff
@@ -2287,6 +2282,505 @@ registry:
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ## Note: notary.signer.podAffinityPreset, notary.signer.podAntiAffinityPreset, and notary.signer.nodeAffinityPreset will be ignored when it's set
+    ##
+    affinity: {}
+    ## @param notary.signer.nodeSelector Harbor Notary Signer Node labels for pod assignment
+    ## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
+    ##
+    nodeSelector: {}
+    ## @param notary.signer.tolerations Harbor Notary Signer Tolerations for pod assignment
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    ##
+    tolerations: []
+    ## @param notary.signer.topologySpreadConstraints Topology Spread Constraints for pod assignment spread across your cluster among failure-domains. Evaluated as a template
+    ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
+    ##
+    topologySpreadConstraints: []
+    ## @param notary.signer.priorityClassName Priority Class Name
+    ## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+    ##
+    priorityClassName: ""
+    ## @param notary.signer.schedulerName Use an alternate scheduler, e.g. "stork".
+    ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+    ##
+    schedulerName: ""
+    ## @param notary.signer.sidecars Add additional sidecar containers to the Harbor Notary Signer pods
+    ## Example:
+    ## sidecars:
+    ##   - name: your-image-name
+    ##     image: your-image
+    ##     imagePullPolicy: Always
+    ##     ports:
+    ##       - name: portname
+    ##         containerPort: 1234
+    ##
+    sidecars: []
+    ## @param notary.signer.initContainers Add additional init containers to the Harbor Notary Signer pods
+    ## Example:
+    ## initContainers:
+    ##   - name: your-image-name
+    ##     image: your-image
+    ##     imagePullPolicy: Always
+    ##     ports:
+    ##       - name: portname
+    ##         containerPort: 1234
+    ##
+    initContainers: []
+    ## @param notary.signer.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the Harbor Notary Signer pods
+    ##
+    extraVolumeMounts: []
+    ## @param notary.signer.extraVolumes Optionally specify extra list of additional volumes for the Harbor Notary Signer pods
+    ##
+    extraVolumes: []
+    ## @param notary.signer.automountServiceAccountToken Automount service account token
+    ##
+    automountServiceAccountToken: false
+  ## Harbor Notary service configuration
+  ##
+  service:
+    ## @param notary.service.ports.server Harbor Notary server service port
+    ## @param notary.service.ports.signer Harbor Notary signer service port
+    ##
+    ports:
+      server: 4443
+      signer: 7899
+
```

Comment:
Why are you adding this section here?

Reply:
Hi @joancafom This is likely an unintended result of a rebase. Thanks for catching this; I'll remove the new lines.

Reply:
lines +2285 to +2783 have been removed.